README.md

```diff
@@ -0,0 +1,39 @@
+<p align="center">
+  <img src="assets/kf_uikit.png" alt="UIKit">
+</p>
+<p align="center">
+  <a herf="https://github.com/kids-first/kf-uikit/blob/master/LICENSE"><img src="https://img.shields.io/github/license/kids-first/kf-uikit.svg?style=for-the-badge"></a>
+  <a href="https://kf-uikit.netlify.com"><img src="https://img.shields.io/badge/storybook-master-ff69b4.svg?style=for-the-badge"></a>
+  <a href="https://circleci.com/gh/kids-first/kf-uikit/tree/master"><img src="https://img.shields.io/circleci/project/kids-first/kf-uikit.svg?style=for-the-badge"></a>
+</p>
+
+Kids First UIKit
+================
+
+The UIKit implements the Kids First Design System which dictates all visual
+styles of the Kids First project.
+Sub projects within Kids First should utilize the UIKit so that all assets
+maintain consistent appearence.
+
+## Setting up in a new project
+
+To get started with a new application using the UIKit in a new React app:
```

Comment:
Depending on the direction we go in we will need to give more info on how the use the `KFThemeProvider` to get started.